cmd/surf/commands/tests/listmodules_test.go

```diff
@@ -51,10 +50,10 @@ func (suite *ListModuleSuite) TestGetAllModules_Execute_Returns_An_Error_If_The_
 	assert.Equal(suite.T(), expectedErr, actualErr)
 }
 
-func aListOfModules(names ...string) interface{} {
-	expected := make(ch360.ModuleList, len(names))
+func aListOfModules(ids ...string) interface{} {
+	expected := make(ch360.ModuleList, len(ids))
 
-	for index, name := range names {
+	for index, name := range ids {
 		expected[index] = ch360.Module{
 			Name: name,
 			ID:   name,
```

Comment:
Not from this PR, I know, but this doesn't look quite right to me... The module name is something like "Currency", while the module ID is something like `waives.currency`. Possibly doesn't matter for tests as perhaps they are just after a set strings and it doesn't matter what the values are?

Reply:
Yeah, it doesn't really matter here - it's just for the test fixture setup.